project-management/src/main/java/life/qbic/projectmanagement/application/measurement/validation/MeasurementValidationResult.java

```diff
@@ -7,50 +7,52 @@
 import java.util.stream.Stream;
 
 /**
- * <b><class short description - 1 Line!></b>
+ * <b>Measurement Validation Result</b>
```

Comment:
I do no think you have to split the validation result in two. You can keep the general validation result as it is only a means to transfer data about the result of a validation. This can be successful or failed and may additionally contain some warnings.